lib/buffer.js

```diff
@@ -624,10 +624,30 @@ function stringSlice(buf, encoding, start, end) {
   throw new ERR_UNKNOWN_ENCODING(encoding);
 }
 
-Buffer.prototype.copy =
-  function copy(target, targetStart, sourceStart, sourceEnd) {
-    return _copy(this, target, targetStart, sourceStart, sourceEnd);
-  };
+function validateOffsetType(val, name) {
+  if (val) {
```

Comment:
We should only use a default in case it's definitely not set. So I would write it as:


```suggestion
  if (val != null) {
```